jedeschule/spiders/rheinland_pfalz.py

```diff
@@ -43,6 +66,18 @@ def parse_school(self, response):
     def normalize(self, item: Item) -> School:
         zip, city = item.get("Anschrift")[-1].split(" ", 1)
         email = item.get("E-Mail", "").replace("(at)", "@")
+
+        kurzbezeichnung = item.get('Kurzbezeichnung')
+        if kurzbezeichnung:
```

Comment:
If you wanted to be really fancy, I think this would be a use case for the walrus operator:
```suggestion
        if kurzbezeichnung := item.get('Kurzbezeichnung'):
```